Service/src/routes/charts.py

```diff
@@ -8,18 +8,21 @@
 from request_validator import validator
 from request_validator.schemas import charts_schema
 from util.error_map import errors
+from util.db_connection import getConfigFilePath
 
 DAY = '1d'
 MONTH = '1m'
 YEAR = '1y'
 
-IEX_DATETIME_FORMAT = '%Y%m%d %H:%M'
+IEX_DATETIME_FORMAT = '%Y-%m-%d %H:%M'
```

Comment:
I'm guessing IEX changed their format then?

Reply:
Seemed like it